docs/reference/commandline/container_run.md

```diff
@@ -326,7 +326,32 @@ are broken into multiple containers, you might need to share the IPC mechanisms
 of the containers, using `"shareable"` mode for the main (i.e. "donor")
 container, and `"container:<donor-name-or-ID>"` for other containers.
 
-### <a name="privileged"></a> Full container capabilities (--privileged)
+### <a name="privileged"></a> Escalate container privileges (--privileged)
+
+The `--privileged` flag gives the following capabilities to a container:
+
+- Enables all Linux kernel capabilities
+- Disables the default seccomp profile
+- Disables the default AppArmor profile
+- Disables the SELinux process label
+- Grants access to all host devices
+- Makes `/sys` read-write
+- Makes cgroups mounts read-write
+
+In other words, the container can then do almost everything that the host can
+do. This flag exists to allow special use-cases, like running Docker within
+Docker.
+
+> **Note**
```

Comment:
I'd be fine with making this a warning. We should warn that a container with `--privileged` is, well, effectively "not containing", and is able to take control of the host. Something like this gives you a `root` shell on the host;

```bash
docker run -it --rm --privileged --pid=host debian nsenter -t 1 -m -u -n -i sh
```

Reply:
Good call. I'll update